### ISSUE TITLE
Implements With Tensorflow

### ISSUE THREAD

**Issue by Hendriyawan** [IoT-gamer/hand_landmarker#2]
Hello, straight to the point, previously I implemented tensorflow to read SIBI sign using mediapipe handlandmark in python, the results were close to accurate, but when I implemented it in this hand landmarker plugin, why were the results inaccurate?

**Comment by IoT-gamer** [comment]
Hello,

The discrepancy is likely due to the preprocessing differences between the Python desktop environment (usually 4:3 aspect ratio) and mobile (usually tall/narrow aspect ratio).

To help debug this, could you clarify a few details about your TensorFlow gesture classification model:

1. **Coordinate Type:** Was it trained on normalized coordinates (0.0 - 1.0 relative to screen) or world coordinates (3D metric space)?

2. **Training Source:** What was the resolution/aspect ratio of the camera used to collect your training data? (e.g., Webcam 640x480).

3. **Preprocessing:** Are you performing any normalization on the coordinates in your Dart code (e.g., centering them on the wrist) before passing them to the model?

**Comment by Hendriyawan** [comment]
Hi, thanks for the reply, I managed to read the model (SIBI Detection), but the problem is that the phone screen must be tilted (landscape). do you have a solution?, thanks...

**Comment by IoT-gamer** [comment]
Do you just need to rotate the coordinates like:
```dart
Landmark rotate90(Landmark point) {
  return Landmark(
    point.y,         // New X
    1.0 - point.x,   // New Y
    point.z,
  );
}
```
you can plot the landmark points for wrist [0], thumb tip [4], index finger tip [8], pinky tip [20] and see if they are what you expect for the model
